services/frontend-service/src/ui/Pages/Home/Home.tsx

```diff
@@ -36,12 +30,9 @@ export const Home: React.FC = () => {
 
     const { app, version } = useReleaseDialog(({ app, version }) => ({ app, version }));
 
-    const envs = useCurrentlyDeployedAt(app, version);
-    const releaseInfo = useReleaseInfo(app, version);
-
     return (
         <main className="main-content">
-            <ReleaseDialog app={app} version={version} release={releaseInfo} envs={envs} />
+            {app && <ReleaseDialog app={app} version={version} />}
```

Comment:
I moved this to the App component so it can always be displayed on any page, as you can see [here](https://github.com/freiheit-com/kuberpult/pull/424/files#diff-0f737d6ef3cd09ae791a8ca647e09c109a1a96eeceb1f55e7bf90899d829f37fR82)

Also why did you change the props for the release dialog?